R/models.R

```diff
@@ -1,9 +1,10 @@
-#' Predict on vector form
+#' Generate predictions for different model classes
 #'
 #' @description Performs prediction of response \code{\link[stats]{lm}}, \code{\link[stats]{glm}},
-#' \code{\link[ranger]{ranger}} and \code{\link[xgboost]{xgboost}} with binary or continuous response.
+#' \code{\link[ranger]{ranger}} and \code{\link[xgboost]{xgboost}} with binary or continuous
+#' response. See details for more information.
 #'
-#' @param x Object of class inheriting from one of the supported models. See details.
+#' @param x Object of class inheriting from one of the supported models.
```

Comment:
I don't understand the sentence. Different suggestion below.
```suggestion
#' @param x Model object for the model to be explained.
```